components/tag/src/stories.tsx

```diff
@@ -13,11 +14,109 @@ export default {
 export const Default: Story = (args) => <Tag {...args} />;
 Default.args = {
   children: 'beta',
-  inactive: false,
 };
 
 export const Inactive: Story = (args) => <Tag {...args} />;
 Inactive.args = {
-  children: 'beta',
-  inactive: true,
+  children: 'inactive',
```

Comment:
this should have triggered a change in visual regression, but for some reason Chromatic didn't pick it up - I'll need to look in to this